preprocessing/test-data/security_manager.py

```
@@ -0,0 +1,27 @@
+def execute_dynamic_code(code):
+    # Risk: Direct use of eval allows execution of arbitrary code
+    return eval(code)
+
+def validate_session(token):
+    # Risk: Hardcoded secret for JWT validation
+    secret = "static_secret_key"
+    try:
+        decoded = jwt.decode(token, secret, algorithms=["HS256"])
+        if "user_id" in decoded:
+            return True
+        return False
```

Ruff (0.8.2)

Return the condition `"user_id" in decoded` directly

Replace with `return "user_id" in decoded`

(SIM103)